src/main/java/TaskList.java

```diff
@@ -0,0 +1,18 @@
+import task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * TaskList is a data structure to hold a list of Tasks.
+ */
+public class TaskList extends ArrayList<Task>{
```

Comment:
Should there be a space before `{`?
I think in general this is also applicable for other structures such as `for`, `while` and `if`.

Reply:
Same sentiments here, perhaps you could have run your code through the style checker again. It will help identify small mistakes like this.